docs/MPURegionsUsageGuide.md

```diff
@@ -31,39 +31,45 @@
 
 ## How to control MPU
 1. **How to add new MPU region**
-	- Logical mapping of MPU regions to their respective memory segments is done statically using below enumeration in common mpu.h header file (os/include/tinyara/mpu.h).
+	- Logical mapping of MPU regions to their respective memory segments is done statically using below structure in common mpu.h header file (os/include/tinyara/mpu.h).
 	```c
-	enum MPU_REG_NUM {
-	#ifdef CONFIG_APP_BINARY_SEPARATION
-	#ifdef CONFIG_SUPPORT_COMMON_BINARY
-	#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
-		MPU_REG_NUM_COM_LIB_TXT,	/* Common Binary */
-		MPU_REG_NUM_COM_LIB_RO,
-		MPU_REG_NUM_COM_LIB_DATA,
-	#else
-		MPU_REG_NUM_COM_LIB,
-	#endif
-	#endif
-	#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
-		MPU_REG_NUM_APP_TXT,		/* Apps */
-		MPU_REG_NUM_APP_RO,
-		MPU_REG_NUM_APP_DATA,
-	#else
-		MPU_REG_NUM_APP,
-	#endif
-	#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
-		MPU_REG_NUM_STK,		/* Stack */
-	#endif
-	#else
-		MPU_REG_NUM_UFLASH,
-		MPU_REG_NUM_URAM,
-	#endif
-		MPU_REG_NUM_MAX
+	struct mpu_usages_s {
+	        uint8_t nregion_board_specific;
+	        uint8_t nregion_common_bin;
+	        uint8_t nregion_app_bin;
+	        uint8_t nregion_stackovf;
+	        uint8_t max_nregion;
 	};
 	```
-	- The ordering of MPU regions in this enum is kept as above taking into consideration below rule:
+	- These regions are initialized to their respective numbers during MPU initialization in mpuinit.c file (arch/arm/src/common/up_mpuinit.c) as below:
+	```
+	void mpu_region_initialize(struct mpu_usages_s *mpu)
+
+	offset += MPU_NUM_REGIONS;
+	mpu->nregion_xxx_xx = offset;
+	```
+	where,
+	 MPU_NUM_REGIONS is set to:
```

Comment:
Please add two spaces at end of the line to give line change.

Reply:
You need to check all of lines where you changed a line. I think you added two spaces here only. Please find your change through `view file`.

Reply:
Modified